File: Pattern/pattern8/pattern.go
package main

import "fmt"

func pattern(x int) {
	for i := 1; i < x; i++ {
		for j := 1; j < i; j++ {
			fmt.Print(" ")
		}
		for k := 1; k <= x-i; k++ {
			fmt.Print("* ")
		}
		fmt.Println("")
	}
}

func main() {
	pattern(7)
}

File: Pattern/pattern8/Pattern.java
package Pattern.pattern8;

public class Pattern {
    public static void main(String[] args) {
        pattern(10);
    }

    public static void pattern(Integer n) {
        for (int i = 0; i < n; i++) {

            for (int k = 1; k <= i; k++) {
                System.out.print(" ");
            }

            for (int j = 1; j <= n - i; j++) {
                System.out.print("* ");
            }

            System.out.println("");
        }
    }
}
